vspace/src/main/webapp/WEB-INF/views/staff/modules/slides/slide.jsp

```diff
@@ -95,14 +119,20 @@ function uploadImage() {
             $(".open").removeClass("open");
             var $imgTag = imageblock.find('img[id]');
             if($imgTag.length == 0){
```

Comment:
what is this check for?

Reply:
I think, it's not being used. I will remove that after testing.